Library/Formula/awscli.rb

```diff
@@ -96,6 +101,9 @@ def install
   end
 
   def caveats; <<-EOS.undent
+    Python >2.7.2 is recommended to prevent urllib3 SSL warnings:
```

Comment:
The only reason is that each time the user will call any `aws` command that talks to the API (which is pretty much any one) will see the warning and may think that awscli is broken. But I'm happy with removing it.